angular.json

```diff
@@ -12,8 +12,7 @@
         "@schematics/angular:application": {
           "strict": true
         },
-        "@schematics/angular:resolver": {
-          "functional": true,
+        "@schematics/angular:directive": {
```

Comment:
Why has this changed? Just interested to know

Reply:
Think the resolver bit must have got deleted when resolving merge conflicts, so I've added that back - changed the directive to not be flat so it, and its spec files sit in the same folder, just like components and services.